docker-compose.auv8.yml

```diff
@@ -263,6 +263,8 @@ services:
     image: ghcr.io/sonia-auv/octopus-telemetry:develop
     container_name: octopus_telemetry
     network_mode: host
+    environment:
+      - REACT_APP_BRIDGE_IP=192.168.0.31
```

Comment:
@alexemdesgagne Shouldn't we place this IP in the launch as an environnement variable?? The end goal for the docker-compose is to have one docker-compose file for the 3 launch files.

Reply:
That's a good point ! I'll fix that today !

Reply:
Done. I used the variable defined in the bash file. Tested on my local machine and working !